src/warnet/deploy.py

```diff
@@ -90,6 +95,30 @@ def deploy_logging_stack(directory: Path, debug: bool):
     return True
 
 
+def deploy_caddy(directory: Path, debug: bool):
+    network_file_path = directory / NETWORK_FILE
+    with network_file_path.open() as f:
+        network_file = yaml.safe_load(f)
+
+    namespace = "warnet-logging"
+    name = "caddy"
```

Comment:
"warnet-logging" should be a constant in `constants.py` imo, and `name` should definitely be coming from the helm chart and not hardcoded here.